python/semantic_kernel/connectors/ai/open_ai/services/open_ai_chat_completion.py

```diff
@@ -267,7 +262,13 @@ async def _send_chat_request(
                 model_args["functions"] = functions
 
         try:
-            response: Any = await openai.ChatCompletion.acreate(**model_args)
+            client = AsyncOpenAI(
```

Comment:
we should probably move this client into the class attributes (including the api details) so that we do not reinitialise the creation, would also make differentiating the init between the openai and azureopenai version easier, both inits end with a initialized asyncopenai client as self.client and then the completions can just use that.